### ISSUE TITLE
Feature request: num downvote / flags included in discussion details

### ISSUE THREAD

**Issue by evm2p** [adcpm/steem#52]
Currently the number of downvotes / flags are not included in the response for a discussion, i.e. when fetching posts. Can you add this?

**Comment by p0o** [comment]
@evm2p It is included. You can access it under `active_votes` object in the returned post data. 

You can see an example of code using `active_votes` to retrieve only flags [here in busy](https://github.com/adcpm/busy/blob/dev/src/app/Menu/MenuPost.js#L36)

**Comment by evm2p** [comment]
Thank you very much @p0o ! Also good luck with busy, I saw your announcement post 😄

**Comment by yamadapc** [comment]
Adding helpers like these could be an idea

**Comment by p0o** [comment]
@yamadapc Yes there are many operations we are doing in apps like busy that could be abstracted away into `steemjs` I think it would be great if we start by designing a better API first.
